idseq_dag/steps/run_srst2.py

```diff
@@ -118,7 +119,7 @@ def normalize_bam_file(self):
     def generate_mapped_reads_tsv(self):
         """Use bedtools to generate a table of mapped reads for each genome in the ARG ANNOT database.
             If a new resistance gene db is used, the .bed file will need to be updated manually."""
-        bed_file_path = fetch_from_s3(self.additional_files["resist_genome_bed"], self.output_dir_local, allow_s3mi=False)
+        bed_file_path = fetch_reference(self.additional_files["resist_genome_bed"], self.ref_dir_local, allow_s3mi=False)
```

Comment:
Here, on the other hand, we are fetching a reference that could well be usefully persisted.   I think!